tests/test_quoting.py

```diff
@@ -473,18 +469,14 @@ def test_unquoter_path_with_plus(unquoter):
 
 
 @given(safe=st.text(), protected=st.text(), qs=st.booleans(), requote=st.booleans())
-@pytest.mark.skipif(
-    not ENABLE_HYPOTHESIS_TESTS, reason="hypothesis tests are not enabled"
-)
+@pytest.mark.hypothesis
 def test_fuzz__PyQuoter(safe, protected, qs, requote):
     """Verify that _PyQuoter can be instantiated with any valid arguments."""
     assert _PyQuoter(safe=safe, protected=protected, qs=qs, requote=requote)
 
 
 @given(ignore=st.text(), unsafe=st.text(), qs=st.booleans())
-@pytest.mark.skipif(
-    not ENABLE_HYPOTHESIS_TESTS, reason="hypothesis tests are not enabled"
-)
+@pytest.mark.hypothesis
```

Comment:
@bdraco apparently, this marker is already being autoapplied:

![Screenshot_2024-10-16-10-41-06-52_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/user-attachments/assets/138fc6b2-15e3-4f8d-ba58-e0e17f6958cd)

*(https://hypothesis.rtfd.io/en/latest/details.html#the-hypothesis-pytest-plugin)*

Reply:
Nice, that makes this a bit cleaner